code/modules/modular_computers/computers/item/modular_computer.dm

```diff
@@ -1,643 +0,0 @@
-// This is the base type that does all the hardware stuff.
```

Comment:
Is the entire contents of this file removed? Why not just delete the file

Reply:
it should be removed. I dont have this file on my local repo